src/python/bot/fuzzers/honggfuzz/engine.py

```diff
@@ -66,6 +68,33 @@ def _find_sanitizer_stacktrace(reproducers_dir):
   return None
 
 
+def _get_reproducer_path(line):
+  """Get the reproducer path, if any."""
+  crash_match = _CRASH_REGEX.match(line)
+  if not crash_match:
+    return None
+
+  return crash_match.group(1)
+
+
+def _get_stats(line):
+  """Get stats, if any."""
+  if not line.startswith(_STATS_PREFIX):
+    return None
+
+  parts = line[len(_STATS_PREFIX):].split()
+  stats = {}
+
+  for part in parts:
+    key, value = part.split(':', 2)
```

Comment:
should we bail out if ':' not in part ? just to avoid exception, maybe log_error ?

Reply:
Done!